lib/couchdb.js

```diff
@@ -747,30 +752,64 @@ CouchDB.prototype.autoupdate = function(models, cb) {
   var self = this;
   async.each(models, autoUpdateOneModel, cb);
 
-  function autoUpdateOneModel(model, callback) {
-    var mo = self.selectModel(model);
-    var properties = mo.mo.properties;
+  function autoUpdateOneModel(model, cb1) {
```

Comment:
@ssh24 I addressed the comment above in my PR, so you don't have to do it again which may run into merge conflict.